packages/affiliates/apps/MultiDappMiningRewards.js

```diff
@@ -0,0 +1,44 @@
+const lodash = require("lodash");
```

Comment:
example of a pipeline that is all built into a single script. it is kind of nice but theres no way to call each stage in the pipeline independently without editing code, which is really annoying if you just want to generate the config file, or verify your github issue template looks correct.